src/auto_archiver/modules/gsheet_db/gsheet_db.py

```diff
@@ -85,6 +86,12 @@ def batch_if_valid(col, val, final_value=None):
 
         gw.batch_set_cell(cell_updates)
 
+    @staticmethod
+    def _get_current_datetime_iso() -> str:
```

Comment:
TODO: there's an open task to make a single method for getting the current datetime. Perhaps this could go in utils instead? https://github.com/orgs/bellingcat/projects/9?pane=issue&itemId=92691996